Source/CarthageKit/Project.swift

```diff
@@ -145,6 +144,23 @@ public final class Project { // swiftlint:disable:this type_body_length
 		let cartfileURL = directoryURL.appendingPathComponent(Constants.Project.cartfilePath, isDirectory: false)
 		let privateCartfileURL = directoryURL.appendingPathComponent(Constants.Project.privateCartfilePath, isDirectory: false)
 
+		// Because the file system Carthage is interacting with may be case sensitive a validation must be
+		// performed to ensure proper file naming. See: https://github.com/Carthage/Carthage/issues/1336
+		func isValidProjectFiles() -> Bool {
```

Comment:
maybe more like `func validateFilenamesOfCartfiles`?

Reply:
Yeah naming this function is a little weird but I do like your suggestion better.